src/server/game/Config/WorldConfig.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2016+ AzerothCore <www.azerothcore.org>
+ */
+
+#ifndef _WORLD_CONFIG_H
+#define _WORLD_CONFIG_H
+
+#include "Common.h"
+#include <unordered_map>
+
+enum class WorldConfigType : uint8
+{
+	GAME_CONFIG_TYPE_BOOL,
+	GAME_CONFIG_TYPE_INT,
+	GAME_CONFIG_TYPE_FLOAT,
+	GAME_CONFIG_TYPE_STRING,
+
+	GAME_CONFIG_TYPE_UNKNOWN
+};
+
+class AC_GAME_API WorldConfig
```

Comment:
I think this is the main reason why build is failing, but cannot figure out what the AC equivalent is

Reply:
Ahahah, ac don't support dynamic lib, need del this macros